db/seeds/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Movies = require('./data/movies');
+
+exports.seed = function (Knex) {
+  return Knex('movies').truncate()
+  .then(() => {
+    return Knex('movies').insert(Movies);
```

Comment:
If an arrow function is only returning one thing we can use the short-hand syntax which makes the code shorter and more readable as it has less indentation and curly braces.
```js
return Knex('movies').truncate()
.then(() => Knex('movies').insert(Movies));
```

Technically we can also do this in the parent function (`seed()`) but because it has a promise chain it doesn't look good and it can be a bit more confusing to read so we tend to do it only for promise chain links only.